docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+
+A primary focus (no pun intended) of @reach/router is to manage focus in client-side routing, out of the box, lifting the onus from devs to manage it from scratch. From the @reach/router documentation:
+
+> Whenever the content of a page changes in response to a user interaction, the focus should be moved to that content; otherwise, users on assistive devices have to search around the page to find what changed–yuck! Without the help of a router, managing focus on route transitions requires a lot effort and knowledge on your part.
+
+> Reach Router provides out-of-the-box focus management so your apps are significantly more accessible without you breaking a sweat.
+
+> When the location changes, the top-most part of your application that changed is identified and focus is moved to it. Assistive devices then announce to the user the group of elements they are now focused on, similarly to how it works when they load up a page for the first time.
+
+The video below demonstrates this focus management:
+
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr">Check out that focus management 😍<br><br>The same code that makes this possible is what makes relative links and embedded routers possible too. <a href="https://t.co/DjqveMfspA">pic.twitter.com/DjqveMfspA</a></p>&mdash; Ryan Florence (@ryanflorence) <a href="https://twitter.com/ryanflorence/status/1002219535921889281?ref_src=twsrc%5Etfw">May 31, 2018</a></blockquote>
+
+In terms of the development experience with Gatsby, this change is mostly under the hood, folded into the implementation of the Gatsby Link component. In terms of usability, accessibility by default is a win for everyone 🙌🏻.
+
+## Migrating from v1 ➡️ v2
+
+For most sites, migrating from v1 to v2 shouldn’t be too painful, but there are a few instances you might want to be aware of. Check out the [v2 migration guide](/docs/migrating-from-v1-to-v2/#migrate-from-react-router-to-reachrouter) for details.
+
+## TLDR;
+
+[Smaller package + better accessibility + simplified APIs](https://github.com/gatsbyjs/gatsby/pull/6918) 👍
+
+We look forward to continuing to work actively with Ryan!
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Reach Router is hitting the big time with <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> adopting it and Nike shipping a site with it <a href="https://t.co/fthOUQ1lJh">https://t.co/fthOUQ1lJh</a>  :D<br><br>I&#39;ll be spending all day Thursday fixing/adding stuff Gatsby needs. AND! Gatsby is sponsoring my time.<br><br>Thanks <a href="https://twitter.com/kylemathews?ref_src=twsrc%5Etfw">@kylemathews</a> and the rest of the team 🙏🏽</p>&mdash; Ryan Florence (@ryanflorence) <a href="https://twitter.com/ryanflorence/status/1042117992140554241?ref_src=twsrc%5Etfw">September 18, 2018</a></blockquote>
+
```

Comment:
Just wondering, will it be obvious to people that reach-router does a lot more (just reading their docs site now). Like "it totally replaces what we were doing before, so we're not losing anything, only gaining something better."

Reply:
Not sure I'd want to put that out there quite like that. It does have some (known, intentional) tradeoffs -- https://reach.tech/router/credits

Reply:
@shannonbux same thing. other edits made -- going to resolve this unless there's further comment.

Reply:
sounds great